### ISSUE TITLE
C# Script autocompletion

### ISSUE THREAD

**Issue by NogradThGin** [Unity3D-Wine-Support/Unity3D-on-Wine#53]
Hi, 

Actually i got a problem with C# autocompletion under native MonoDevelop. I run Unity 5 on ArchLinux x64 (Updated 02/06/2015) with native MonoDevelop 5.0. I searched during 2 days on Internet, but found nothing. I can't use built-in monodevelop too because the keyboard layout isn't good (i'm french so my Keyboard layout is fr_FR.UTF-8)

**Comment by DoctorJellyface** [comment]
Did you use the scripts in this repo?

**Comment by NogradThGin** [comment]
Yes,

I installed Unity3D with wine 1.7.38 compiled for it. 
https://aur.archlinux.org/packages/wine-unity3d-git/ (i use this)

**Comment by DoctorJellyface** [comment]
Hm. (pro-tip, you can use [`wine-staging`](https://aur.archlinux.org/packages/wine-staging/) (formerly `wine-silverlight` to get OpenGL mode to work, plus some more fixes).

What is the **wine** path to your project?

Also, try opening Monodevelop manually and opening the project file through it.

**Comment by NogradThGin** [comment]
Thanks for tip but Unity already work on OpenGL mode :p My wine path is ~/.wine.

I manually launched MonoDevelop and load my scripts but i got the same result. I think i don't explained correctly my problem (my bad sorry), the problem is i got autocompletion work, but i can't see anything about Unity. For example i can see System.Collections in auto completion (using System.Collections;) but i can't see UnityEngine or another thing related to Unity

**Comment by DoctorJellyface** [comment]
No, I meant the Wine path to your project, not the wineprefix. It's in a Windows `C:\Path\To\Project` format.

What do you mean by Unity working with OpenGL? As far I know it does not work in a unpatched Wine. Last time I tested was on 1.7.44. I only suggested wine-staging because it has some nifty features and a repo, besides the OpenGL mode working.

When you manually launched it from MonoDevelop, did you have Unity open in the background?

**Comment by NogradThGin** [comment]
Ah. I think you talk about path to my project, it's "~/.wine/drive_c/users/vince/Mes\ documents/GameScripts/"

The Wine version i use is specially patched and compiled (on my PC) for Unity.

And yes, when i started Native MonoDevelop manually, Unity is opened.

**Comment by DoctorJellyface** [comment]
No, I meant the **Wine** (windows) path. The one that's written under the project name in the opening window.
